roles/openshift_metrics/templates/hawkular_metrics_schema_job.j2

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: hawkular-metrics-schema
+  labels:
+    metrics-infra: hawkular-metrics
+    name: hawkular-metrics-schema
+spec:
+  template:
+    spec:
+      version: v1
+      metadata:
+        labels:
+          metrics-infra: hawkular-metrics
+          #name: hawkular-metrics
+      containers:
+      - name: hawkular-metrics-schema
+        image: {{openshift_metrics_image_prefix}}metrics-schema-installer:{{openshift_metrics_image_version}}
```

Comment:
This image doesn't exist, it resolves to `docker.io/openshift/origin-metrics-schema-installer:v3.9` for Origin

Reply:
@rubenvp8510 I think this is blocking the merge of this PR, have you had a chance to check on this?

Reply:
Yes, the scripts for push the metrics-schema image to dockerhub are already updated on master, but the image is not there, still investigating what I need to do or who I need to talk with in order to push the images.

Reply:
We are publishing images for master, is that not appropriate?